.drone.yml

```diff
@@ -254,7 +254,8 @@ matrix:
       TEST_SUITE: owncloud-coding-standard
 
     # phan
-    - OC_VERSION: daily-master-qa
+    - PHP_VERSION: 7.1
```

Comment:
@patrickjahns I need to have "server" code available so that ``phan`` knows about the core code/classes etc.
When I put ``NEED_SERVER`` for this matrix entry, then the ``NEED_SERVER`` pipeline step needs ``PHP_VERSION`` to be set.

When the first ``phan-70`` pipeline step runs, it does ``composer install`` and gets the needed ``phan`` dependencies... and that is all good.

When ``phan-71`` and ``phan-72`` run, they do not go through a "full" ``composer install`` 

It all passes, but I am not that confident about whether we are really doing effective tests for each of PHP 7.0 7.1 and 7.2

An alternative is to go back to having 3 matrix entries, 1 for ``phan`` on each PHP version. That will ensure the server and ``phan`` and everything are really using the requested PHP version.

Thoughts?

Reply:
The dependencies from phan and core are completely segregated. That is what `composer-bin` ensures.
As phan itself is having a platform limited to php 7.0 ( https://github.com/phan/phan/blob/master/composer.json#L20-L22 ) - we are fine either way on installing it on any php version > 7.0

Phan will evaluated the AST from the code in question in the php runtime it currently is executed. So we need to execute it with every php version we want it to be evaluated against

Reply:
> Phan will evaluated the AST from the code in question in the php runtime it currently is executed. So we need to execute it with every php version we want it to be evaluated against

Adding to that - the core dependencies should also not change per php version - so it does not matter. And as stable10 is also php7.2 compatible - we might always use php7.2 for the core install step ( no matter if it is master or stable10 )

Reply:
Bumped PHP_VERSION here from 7.1 to 7.2

Reply:
I was wrong - `7.1` is/was fine ( thought that master required min 7.2 🙈 ). 
But `7.2` doesn't hurt either way - it shouldn't make a difference in this matter